src/Fields/Fields.jl

```diff
@@ -437,11 +437,11 @@ ClimaCore.Fields.truncate_printing_field_types() = true
 =#
 truncate_printing_field_types() = false
 
-function Base.show(io::IO, ::Type{T}) where {T <: Fields.Field}
```

Comment:
This is probably because `Union{} <: Fields.Field`: I've seen this in other cases.